.github/workflows/test-all.yml

```diff
@@ -1,11 +1,7 @@
 name: Run All Tests
```

Comment:
Since the other workflow is renamed to "Test Modified Packages", how about also renaming this one to "Test All Packages"?

Reply:
+1

Reply:
Done.